server/lib/adapters/changes.ex

```diff
@@ -6,24 +6,52 @@ require Protocol
 defmodule Realtime.Adapters.Changes do
   defmodule(Transaction, do: defstruct([:changes, :commit_timestamp]))
 
-  defmodule(NewRecord,
-    do: defstruct([:type, :record, :schema, :table, :columns, :commit_timestamp])
-  )
+  defmodule NewRecord do
+    @derive {Jason.Encoder, except: [:is_rls_enabled, :users]}
+    defstruct [
+      :columns,
+      :commit_timestamp,
+      :schema,
+      :table,
+      :record,
+      :users,
+      :type,
+      is_rls_enabled: true
+    ]
```

Comment:
when `walrus` is ready to send meaningful `error`  message, like[ database change was bigger than n mb](https://github.com/supabase/walrus/issues/11), then we can add an additional `error` key to the struct